src/main/java/htsjdk/variant/variantcontext/filter/VariantContextFilter.java

```diff
@@ -24,20 +24,14 @@
 package htsjdk.variant.variantcontext.filter;
 
 import htsjdk.variant.variantcontext.VariantContext;
+import java.util.function.Predicate;
 
 /**
  *
- * API for filtering VariantContexts
+ * API for filtering VariantContexts.
  *
  * @author Yossi Farjoun
  *
  */
-public interface VariantContextFilter {
-    /**
-     * Determines whether a VariantContext matches this filter
-     *
-     * @param record the VariantContext to evaluate
-     * @return true if the VariantContext matches the filter, otherwise false
-     */
-    boolean test(VariantContext record);
+public interface VariantContextFilter extends Predicate<VariantContext> {
```

Comment:
This is OK but note that it's not required as you can use the original as `stream.filter(f::test)`. But this form lets you write `stream.filter(f)` which is a bit shorter.